features/cli-cache.feature

```diff
@@ -0,0 +1,19 @@
+Feature: CLI Cache
```

Comment:
The tests currently only test the output of the command, not whether it actually does what it pretends to do.

Testing should maybe download 2 or 3 files, verify their presence in the cache, and then verify that a cache clear/prune actually removes some or all of them again.

Reply:
Yeah I know 😞

Can you point me to the right direction, how I should do these downloads, etc.? I'm sure there are some helper methods for that I don't yet know about.

Reply:
I'll look into how best to do this.

Reply:
So, the general approach should be the following:
1. Initialize a separate, empty cache dir:
   ```
   Given an empty cache dir
   ```
2. The cache dir that was generated is now available from within the tests under the `SUITE_CACHE_DIR` variable
3. Make some operation that should cache something
4. Verify that a file was created under `SUITE_CACHE_DIR`
5. Clear the cache
6. Verify that the file is gone under `SUITE_CACHE_DIR`

Reply:
> Make some operation that should cache something

Reply:
> Make some operation that should cache something

Are things like the language command available in the tests? Or perhaps the command to download WP core?

Reply:
Yes, they should be available with a simple `When I run wp something something`.

Reply:
Maybe stick with Core, as I want to reduce the amount of inter-dependencies that the tests introduce.
I think you can force multiple versions of Core to be downloaded, so this will allow you to test the `prune` as well.